ui/src/onboarding/containers/SigninPage.tsx

```diff
@@ -67,18 +68,18 @@ class SigninPage extends PureComponent<Props, State> {
         loading={this.state.status}
         spinnerComponent={<TechnoSpinner />}
       >
-        <SplashPage>
-          <Panel className="signin-panel">
+        <FunnelPage className="signin-page">
+          <Panel className="signin-page--panel">
             <Panel.Body alignItems={AlignItems.Center}>
-              <SplashPage.Logo />
-              <SplashPage.Header title="InfluxData" />
+              <div className="signin-page--cubo" />
+              <InfluxDBCloudLogo cloud={false} className="signin-page--logo" />
```

Comment:
what is `cloud={false}` we have a `CLOUD` global variable that denotes whether we're in a cloud context or not.

Reply:
that prop controls whether the word "Cloud" renders in the image. https://influxdata.github.io/clockface/?path=/story/graphics-brand-logos--influxdb-cloud

In this instance I left it `false` cuz this page is OSS only